views/components/Maps/Maps.tsx

```diff
@@ -87,10 +89,10 @@ export default class Maps extends React.Component<IMapsProps, IMapsState> {
                         </LayerGroup>
                     </LayersControl.Overlay>
                     <LayersControl.Overlay name="Local Path">
-                        <Polyline pathOptions={{ color: 'black' }} positions={[/* Add Local Path here*/]} />
+                        <Polyline pathOptions={{ color: 'black' }} positions={[this.props.localPath]} />
                     </LayersControl.Overlay>
                     <LayersControl.Overlay name="Global Path">
-                        <Polyline pathOptions={{ color: 'black' }} positions={[/* Add Global Path here */]} />
+                        <Polyline pathOptions={{ color: 'black' }} positions={[this.props.globalPath]} />
```

Comment:
Let's add `opacity: 0.25` to `pathOptions` for global path. This will help us distinguish between global path and local path

Reply:
done